nextcloud_update.sh

```diff
@@ -784,7 +784,12 @@ then
         fi
         if [ -n "${APPSTORAGE[$app]}" ] && [ "${APPSTORAGE[$app]}" != "yes" ] && is_app_enabled "$app"
         then
-            nextcloud_occ_no_check config:app:set "$app" enabled --value="${APPSTORAGE[$app]}"
```

Comment:
This is what I mean with: 
> (we already reinstall disabled apps there, and also set it to disabled but don't use the official app:disable command there.)

(initially we didn't differentiate between the value beeing set to an array of enabled groups or to `no`. This PR fixes this by using the official command to disable the app if it was disable before the update was run) But it already does kind of the same: reinstalling and disabling apps that were disabled before the update was executed